Index: cmake/modules/FindNETCDF.cmake
===================================================================
--- cmake/modules/FindNETCDF.cmake	(revision 18324)
+++ cmake/modules/FindNETCDF.cmake	(working copy)
@@ -75,11 +75,11 @@
 
 find_path (NETCDF_INCLUDE_DIR netcdf.h
 	HINTS
-	${_netcdf_includepath}
 	${NETCDF_DIR}
 	${NETCDF_ROOT}
 	$ENV{NETCDF_DIR}
 	$ENV{NETCDF_ROOT}
+	${_netcdf_includepath}
 	PATH_SUFFIXES
 	include/netcdf
 	include/netcdf-4
@@ -96,11 +96,11 @@
 find_library (NETCDF_LIBRARY
 	NAMES netcdf
 	HINTS
-	${_netcdf_libpath}
 	${NETCDF_DIR}
 	${NETCDF_ROOT}
 	$ENV{NETCDF_DIR}
 	$ENV{NETCDF_ROOT}
+	${_netcdf_libpath}
 	PATH_SUFFIXES lib
 	PATHS
 	/sw
